docs/docs/docs/getting-started/installation.md

```
@@ -192,35 +192,19 @@ Click on the image below to play the video.
-
-Follow these steps to have the best experience
-
-#### Using the CLI
+### Setup: Using the CLI
 
 These steps are specific to Linux. You will need to modify them accordingly for other operating systems
 
 1. Install `docker` and ensure that the daemon is running.
-1. Install the PostgreSQL database
+1. Install the PostgreSQL database.
    ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)